lib/warder/coffee_lint_runner.rb

```diff
@@ -0,0 +1,37 @@
+module Warder
+  # responsible for run coffeescript style guide validation
+  class CoffeeLintRunner < Runner
+    require 'warder/coffee_lint_runner/lint_runner'
+    require 'warder/coffee_lint_runner/message_formatter'
+
+    CLI_OPTION = 'g'
+    CLI_FULL_OPTION = 'coffee-lint'
+    DESCRIPTION = 'Run coffeescript style guide validation'
+    COMMAND_NAME = 'coffeelint'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.